js/src/wrappers/anthropic.ts

```diff
@@ -1,13 +1,36 @@
-import Anthropic from "@anthropic-ai/sdk";
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { Span, startSpan } from "../logger";
 import { SpanTypeAttribute } from "@braintrust/core";
 import { filterFrom, getCurrentUnixTimestamp } from "../util";
 
-export function wrapAnthropic(anthropic: Anthropic): Anthropic {
-  return anthropicProxy(anthropic);
+/**
+ * Wrap an `Anthropic` object (created with `new Anthropic(...)`) to add tracing. If Braintrust is
+ * not configured, this is a no-op
+ *
+ * Currently, this only supports the `v4` API.
+ *
+ * @param anthropic
+ * @returns The wrapped `Anthropic` object.
+ */
+export function wrapAnthropic<T extends object>(anthropic: T): T {
+  const anthropicUnknown: unknown = anthropic;
```

Comment:
nit: i feel like this section would be more readable with a shorter variable like e.g. `a` or `au`. The length of the method makes the short variable name fine.

Reply:
happy to!